packages/expo-maps/src/index.ts

```diff
@@ -23,6 +23,8 @@ export namespace GoogleMaps {
   export type MapUiSettings = GoogleTypes.MapUiSettings;
   export type MapProperties = GoogleTypes.MapProperties;
   export type MapProps = GoogleTypes.MapProps;
+
+  export type MapViewType = GoogleTypes.MapViewType;
```

Comment:
```suggestion
  export type MapView = GoogleTypes.MapViewType;
```
This might look better in the public api? wdyt?